src/app/shared/models/language/language.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface Language {
```

Comment:
I think it should be call like: DotLanguage

Reply:
They could extends from Contentlet or something like that in order to keep a common interface to apply common methods

Reply:
Not really a Language is not a Contentlet